_data/simple-icons.json

```
@@ -12678,6 +12678,31 @@
             "hex": "EA5906",
             "source": "https://webdriver.io/docs/api/"
         },
+        {
+            "title": "Webex",
+            "hex": "FFFFFF",
+            "source": "https://www.webex.com/",
+            "guidelines": "https://resources.webex.com/webex/brand-exchange-collection",
+            "license": {
+                "type": "custom",
+                "url": "https://www.cisco.com/c/en/us/about/legal/trademarks.html"
+            },
+            "aliases": {
+                "aka": [
+                    "Cisco Webex",
+                    "Webex by Cisco"
+                ],
+                "old": [
+                    "WebEx",
+                    "Spark",
+                    "Cisco Spark",
+                    "WebEx Teams",
+                    "Webex Teams",
+                    "WebEx Meetings",
+                    "Webex Meetings"
+                ]
+            }
+        },
```

```suggestion
        },
```

This section is not for keywords. Please remove these.

---

I have completely removed the `aliases` key. However, the the interactive tool suggested adding these? 
Originally, Webex was known as Spark and "[Webex Teams](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQKWB8rkyLWiba0Q6lREd5CwrlazCPwLy2jTuMEGLia1g&s)" and "[Webex Meeting](https://pbs.twimg.com/media/EV-lq58WsAIimFV?format=jpg&name=small)" were two separate applications (each with their own icons).